gpos/xml/scap/ssg/content/ssg-chainguard-gpos-ds.xml

```diff
@@ -188,238 +189,264 @@
+                            passwords are not found on the list of commonly-used, expected, or
+                            compromised passwords in IA-5 (1) (a).</ns0:title>
+                        <ns0:description xml:lang="en">Password-based authentication applies
+                            to passwords regardless of whether they are used in single-factor or
+                            multifactor authentication. Long passwords or passphrases are preferable
+                            over shorter passwords. Enforced composition rules provide marginal
+                            security benefits while decreasing usability. However, organizations may
+                            choose to establish certain rules for password generation (e.g., minimum
+                            character length for long passwords) under certain circumstances and can
+                            enforce this requirement in IA-5(1)(h). Account recovery can occur, for
+                            example, in situations when a password is forgotten. Cryptographically
+                            protected passwords include salted one-way cryptographic hashes
+                            of passwords. The list of commonly used, compromised, or expected
+                            passwords includes passwords obtained from previous breach corpuses,
+                            dictionary words, and repetitive or sequential characters. The list
+                            includes context-specific words, such as the name of the service,
+                            username, and derivatives thereof.</ns0:description>
+                        <ns0:rationale>The container image is built with no local user accounts other
+                            than a locked root. All interactive or programmatic access to the workload
+                            is handled by the container-orchestration platform, which enforces its
+                            own credential hygiene and MFA controls. The OS inside the container
+                            never receives, stores, or validates passwords and no password database
+                            exists against which to apply bad-password screening, forced post-recovery
+                            changes, passphrase-length rules, or password-generation aids.
+
+                            For more information on inherited controls for containers see the DISA
+                            Container Hardening Process Guide.</ns0:rationale>
+                        <ns0:platform idref="cpe:/o:notapplicable" />
+                    </ns0:Rule>
+                </ns0:Group>
+                                <ns0:Group id="xccdf_mil.disa.stig_group_V-263654">
+    <ns0:Rule id="xccdf_mil.disa.stig_rule_SV-263654r1_rule" selected="true" severity="medium">
+                    <ns0:version>SRG-OS-263654-GPOS-263654</ns0:version>
```

Comment:
Unfortunately, I don't this is quite right. The vulnerability id is not directly mapped into the SRG version id; e.g. this is for `V-263654` which maps to this: https://stigaview.com/products/srg-gpos/v3r2/SRG-OS-000720-GPOS-00170/ so I *think* the `ns0:version` ought to be `SRG-OS-000720-GPOS-00170`

Reply:
Nice, good catch. I'll get this updated.

Reply:
Updated in [`e78d74e` (#20)](https://github.com/chainguard-dev/stigs/pull/20/commits/e78d74e0a2ab49c4884ab06fcc50cb44cb8f00fe).